### ISSUE TITLE
Parsing and saving the outputs into a folder that doesn't exist

### ISSUE THREAD

**Issue by tommywalkie** [tommywalkie/excalidraw-cli#9]
Contextually, I ran into an issue where I wanted to parse some Excalidraw files from a certain place `somewhere/` and then save the outputs in a directory, let's say `somewhere/else/`.

```sh
# Save outputs in a directory that doesn't exist
excalidraw-cli somewhere/ somewhere/else/
```

Currently if the directory `else/` doesn't exist, `excalidraw-cli` considers the parameter value as a file name, so it will save the output into a single file named `somewhere/else.png`, this actually makes sense when aware about it, but may confuse unaware people for hours.

I'm considering adding a warning in case the `INPUT` parameter value is about a directory, but the `OUTPUT` parameter value is a directory that doesn't exist. Or maybe throw an `Error`.

**Comment by lucasavila00** [comment]
My 2 cents:

> Currently if the directory else/ doesn't exist, excalidraw-cli considers the parameter value as a file name

If the string has no "extension" in it (it doesn't end with .png) then it's a folder and not a file.

I think that the CLI shouldn't treat `somewhere/else/` or `somewhere/else` as a file, ever.

`somewhere/else.png` is a file, though

**Comment by tommywalkie** [comment]
@lucasavila00 That's a good point. I should consider `OUTPUT` parameter as a directory entry by default, **if there is such extension, then we expect a file as output**.

**Comment by lucasavila00** [comment]
> Consider OUTPUT parameter as a directory entry by default, if there is such extension, then we expect a file as output.

@tommywalkie I'd go with this heuristic for sure.

**Comment by tommywalkie** [comment]
This may temporarily disable custom output file name — but there is always room for a flag, like `--outputFile` 🤔

**Comment by lucasavila00** [comment]
Currently what's the reasoning of providing an input folder and an output file at the same time?

Does the CLI concatenate all the folders .pngs into the same file?

**Comment by tommywalkie** [comment]
> what's the reasoning of providing an input folder and an output file at the same time?

This is why some testing still needs to be done. 😁 
Implementing the mentionned heuristic should hopefully end this confusion.

**Comment by tommywalkie** [comment]
Fixed as of `0.4.2` and https://github.com/tommywalkie/excalidraw-cli/commit/fdc00aaece43466aec430675d267533d9897d9fe !

**Review and notable changes** :
- **IMPORTANT !** `excalidraw-cli my_diagram.excalidraw` will now output `my_diagram.png` (no more using `.excalidraw.png` file extension !)
- `excalidraw-cli input some/dir/that/doesnt/exist` will create the output directory if doesn't exist
- `excalidraw-cli folder some-file.png` will stop the CLI and output an error ⬇️ 

```
Cannot process the request.
You passed a folder (.\folder\) as input, and a single file (some-file.png) as output.`)
```
